src/model/settings.rs

```diff
@@ -13,7 +13,14 @@ pub const LOGGING_ENABLED: &str = "logging_enabled";
 pub const TTS_ENABLED: &str = "tts_enabled";
 pub const MOUSE_ENABLED: &str = "mouse_enabled";
 pub const SAVE_HISTORY: &str = "save_history";
-pub const SETTINGS: [&str; 4] = [LOGGING_ENABLED, TTS_ENABLED, MOUSE_ENABLED, SAVE_HISTORY];
+pub const CONFIRM_QUIT: &str = "confirm_quit";
+pub const SETTINGS: [&str; 5] = [
+    LOGGING_ENABLED,
+    TTS_ENABLED,
+    MOUSE_ENABLED,
+    SAVE_HISTORY,
+    CONFIRM_QUIT,
```

Comment:
Can you default this to true a few lines below please? I think most people will want it.

Reply:
Changed in 3b2a80a150bd7b4296247e3862310b46c0ed1599

Reply:
Thanks